goss-testing/scripts/python/check_for_unused_drives.py

```diff
@@ -100,11 +101,12 @@ def parse_args():
     logger.debug("Parsing command line arguments: {}".format(sys.argv))
     args = parser.parse_args()
 
-    hw_type = hw_type_map[args.hw_type]    
+    hw_type = hw_type_map[args.hw_type]
+    n_storage_nodes = args.num_storage_nodes
     min_expected_osds = min_osds_per_storage_node[hw_type] * args.num_storage_nodes
     max_expected_osds = max_osds_per_storage_node[hw_type] * args.num_storage_nodes
     logger.info("Based on hardware type ({}) and number of storage nodes ({}): min_expected_osds = {}, max_expected_osds = {}".format(hw_type, args.num_storage_nodes, min_expected_osds, max_expected_osds))
-    return min_expected_osds, max_expected_osds
+    return min_expected_osds, max_expected_osds, n_storage_nodes
```

Comment:
you could just return args.num_storage_nodes.   also, we can prob think about making that a check.  like what is expected vs what is actual

Reply:
return args.num_storage_nodes

Reply:
return args.num_storage_nodes